fixieai/cli/agent/commands.py

```diff
@@ -15,6 +16,7 @@
 from fixieai import constants
 from fixieai.cli.agent import agent_config
 from fixieai.cli.agent import loader
+from fixieai.cli.agent import tunnel as tunnel_
```

Comment:
bleah :)

Reply:
I never know what the right convention is for these...